test-target/local-pod-under-test.yaml

```diff
@@ -20,7 +20,7 @@ spec:
       annotations:
         test-network-function.com/container_tests: '["PRIVILEGED_POD","PRIVILEGED_ROLE"]' # optional
         test-network-function.com/defaultnetworkinterface: '"eth0"' # optional
-        k8s.v1.cni.cncf.io/networks: {{ MULTUS_ANNOTATION }}
+        k8s.v1.cni.cncf.io/networks: '{{ MULTUS_ANNOTATION }}'
```

Comment:
Interesting, all of the other Jinja template style vars don't have quotes?  This must have been what I was missing.

Reply:
No need quotes:
 https://stackoverflow.com/questions/19109912/yaml-do-i-need-quotes-for-strings-in-yaml

Reply:
yep, the quotes are required